src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const allDaysPrice = days * pricePerDay;
+
+  if (days >= longTerm) {
+    return allDaysPrice - longTermDiscount;
+  } else if (days >= shortTerm) {
```

Comment:
Prefer if with return over if else to simplify later conditions.
DON'T add else after if with return - the code after it won't be executed anyway

Reply:
fixed